### ISSUE TITLE
Masking base R and stats

### ISSUE THREAD

**Issue by RaphaelS1** [alan-turing-institute/distr6#124]
This is a decision to be made ASAP as it affects all documentation, tutorials etc.
Currently we mask functions in the base and stats packages. These functions include: `cor, power, sd, var, isSymmetric, mode, truncate, union`. We have four choices:

1. Continue to mask them
2. Don't mask them and instead append each method with '.Distribution' e.g. `sd.Distribution` but these are not generics so the user would have to call `sd.Distribution` not `sd`
3. Don't use R62S3
4. Change function names, e.g.:
   1. sd -> stdev or standardDeviation
   2. var -> variance
   3. isSymmetric -> isASymmetric
   4. mode -> modal
   5. truncate -> truncates (but then we would have to change huberize -> huberizes)
   6. union -> union.Set (most people will use `+`)

**Comment by fkiraly** [comment]
`sd` and `var` are very very common and who knows what kind of unintended interaction effects that will generate - good catch.

I'd prefer renames as in no.4, since it seems the most user friendly to me.

**Comment by RaphaelS1** [comment]
I have changed `sd, var, union`. `isSymmetric` was a generic so this was solved. As for `truncate` (which is used to truncate open connections) I think that this is is rare and advanced enough that anyone who needs it will be comfortable using `base::truncate`. As for `mode` (which is my least favourite function in all of R) I am happy to mask it for the same reason but could also easily change to `modal`.
